### DISCUSSION TITLE
Developer Meeting 13th November - Radio link performance metrics

### DISCUSSION THREAD

**Discussion by knmcguire** [bitcraze/discussions#1642] [Announcements]
![image](https://github.com/user-attachments/assets/6cd92572-5c1b-4c6f-8191-6dd4d316da36)

 This time we will be talking about radio link performance metrics and what we are working on at the moment to improve the perception of communication quality.

Date: Wednesday 13th of October
Time: 15:00 (3 pm) Central European Time ([Timebuddy timezone reference](https://www.worldtimebuddy.com/)
Meeting link: https://meet.google.com/cra-ownj-ufy
Speakers: @gemenerik 

Let's get some better perspective on how the communication quality is evaluated! 

Based on this blogpost: https://www.bitcraze.io/2024/10/link-performance-metrics/

**Comment by gemenerik** [comment]
To test the upcoming metrics check out the draft prs https://github.com/bitcraze/crazyflie-lib-python/pull/494 and https://github.com/bitcraze/crazyflie-clients-python/pull/732.
Please share your thoughts and opinions.

Recording and slides can be found [on YouTube](https://youtu.be/8irYB_knt9w). 
[![](https://markdown-videos-api.jorgenkh.no/youtube/8irYB_knt9w)](https://youtu.be/8irYB_knt9w)
